lib/15_in_words.rb

```diff
@@ -0,0 +1,75 @@
+class Fixnum
+  def in_words
+    hash = {
+      0 => nil,
+      1 => "one",
```

Comment:
Constants like this hash should use `:symbols` instead of `"strings"` for their values as a general rule. This is a much more efficient use of memory in Ruby.